CDP4Authentication/Contracts/AuthenticatorPlugin.cs

```diff
@@ -6,11 +6,12 @@
 
 namespace CDP4Authentication.Contracts
 {
+    using Newtonsoft.Json;
```

Comment:
sort order is:

System.* first, then any other using statements in alphabetical order